osf/models/base.py

```diff
@@ -284,8 +284,8 @@ class GuidVersionsThrough(BaseModel):
 
     created = NonNaiveDateTimeField(db_index=True, auto_now_add=True)
     referent = GenericForeignKey()
-    content_type = models.ForeignKey(ContentType, null=True, blank=True, on_delete=models.CASCADE)
-    object_id = models.PositiveIntegerField(null=True, blank=True)
+    content_type = models.ForeignKey(ContentType, null=True, blank=True, on_delete=models.CASCADE, db_index=True)
+    object_id = models.PositiveIntegerField(null=True, blank=True, db_index=True)
```

Comment:
```suggestion
    object_id = models.PositiveIntegerField(null=True, blank=True)
```
delete db_index for ForeignKey, as it is generated automatically